docs/en/ingest-management/migrate-beats-to-agent.asciidoc

```diff
@@ -0,0 +1,513 @@
+{fleet} (or for advanced use cases, specified in a standalone agent policy). Any
+processors you defined previously in the {beats} configuration need to be added
+to an integration policy; they cannot be defined in the {beats} configuration.
+
+IMPORTANT: Globally-defined processors are not currently supported by {agent}.
+You must define processors in each integration policy where they are required.
+
+To add processors to an integration policy:
+
+//REVIEWERS: I noticed that some integrations (like the System integration) do
+//not provide a field for setting processors. Is this intentional? If so, what
+//can I say to explain to users which integrations (or data streams) allow you
+//to add processors?
+
+. In {kib}, got to *{fleet} > Agent policies*, and click the policy name to view
+its integration policies.
+
+. Click the name of the integration policy to edit it.
+
+. Click the down arrow next to enabled streams, and under *Advanced options*,
+add the processor definition. The processor will be applied to the data set
+where it's defined.
++
+[role="screenshot"]
+image::images/migration-add-processor.png[Screen showing how to add a processor to an integration policy]
++
+For example, the following processor adds geo-specific metadata to host events:
++
+[source,yaml]
+----
+- add_host_metadata:
+    cache.ttl: 5m
+    geo:
+      name: nyc-dc1-rack1
+      location: 40.7128, -74.0060
+      continent_name: North America
+      country_iso_code: US
+      region_name: New York
+      region_iso_code: NY
+      city_name: New York
+----
+
+In {kib}, look at the data again to confirm it contains the fields you expect.
+
+[discrete]
+== Preserve raw events
+
+In some cases, {beats} modules preserve the original, raw event, which consumes
+more storage space, but may be a requirement for compliance or forensic use
+cases.
+
+In {agent}, this behavior is optional and disabled by default.
+
+If you must preserve the raw event, edit the integration policy, and for each
+enabled data stream, click the *Preserve original event* toggle.
+
+[role="screenshot"]
+image::images/migration-preserve-raw-event.png[Screen showing how to add a processor to an integration policy]
+
+Do this for every data stream with a raw event you want to preserve.
+
+[discrete]
+== Migrate custom dashboards
+
+Elastic integration packages provide many assets, such as pre-built dashboards,
+to make it easier for you to visualize your data. In some cases, however, you
+might have custom dashboards you want to migrate.
+
+Because {agent} uses data streams, the fields exported by an {agent} are
+slightly different from those exported {beats}. Any custom dashboards that
+you created for {beats} need to be modified or recreated to use the new fields.
+
+//REVIEWERS: Sounds like Beats is moving to data streams, too. Does this whole
+//section only apply to Beats 7.17 and earlier? Should I mention that now?
```

Comment:
yes but they are not writing to logs* or metric* - but let me double check on that.

Reply:
You're correct. Beats use the same naming convention (metricbeat-* etc).  I found this out after I drafted the original topic and dug into Beats 8.0 a bit further. I'll just qualify the statement to say "different data streams" since in 8.0 agent and Beats both use data streams.